### PR TITLE
Feature/user profile page

### PR DESCRIPTION
**Pull request by serpient**
gif

# Things Done
- Profile layout with Card and Citation component
- component is hooked up to call API for participant info. 90% there for updating participant info as well.
- Imported Dynamic forms and Intake form 
- Setup basic Citations form and styling
- test units for Card and Profile component
- add .env file so everyone can use absolute imports
- add responsive design to profile / card / citation component.

# Bugs / Blockers
- Updating Participant data receives an error from the backend. Need to figure out what the error is about. The  PUT route will also need to return the updated user data so I can re-sync in the component.
- need a list of court code and citation status options
- need a list of common violations

# Things To Do 
- add work sans font if available

### Any component with API calls
- add an error component to show error messages to users.
- use loader components as necessary

### Complete Card component
- Ensure updating user data works.
- adjust name row to be responsive as well
- finish the rest of the test units

### Complete Citations component
- Add a list of Court Codes and Statuses options to the form QA.
- Add new question type for Violations section as its a custom style/output. 
- Adjust citation submit / edit / delete button per mockup.
- Hook up citation component with User Citation data; allow for adding new citations, editing of old ones, and deleting any as necessary. 
- Ensure selecting multiple items in dropdown works.
- Add test units for Citations.

### Complete Status Component
- need a list of status options
- Add Status component

### CODE REVIEW

**Comment by mitchellwandrews** [comment]
@serpient I'll pull this down and look at the participants PUT bug before monday. We also do have a citations table
`   id SERIAL PRIMARY KEY,
   citation_number VARCHAR(50),
   court_code VARCHAR(50),
   violation_number VARCHAR(50),
   citation_status VARCHAR(20),
   created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
   updated_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
   participant_id INTEGER,
   FOREIGN KEY (participant_id) REFERENCES participants (id)`

**Comment by serpient** [comment]
perfect! sorry, yeah i just saw the Citations table -- thanks. 

And I couldn't figure out how to get the server to print out console.logs // i may have to see how you or tien does it when I next see you guys!

**Comment by mitchellwandrews** [comment]
@serpient added a commit to this branch for 

> * Updating Participant data receives an error from the backend. Need to figure out what the error is about. The  PUT route will also need to return the updated user data so I can re-sync in the component.

File: main-app/client/src/get_uri.js [hunk 1]
```diff
@@ -1 +1 @@
-export const API_ENDPOINT = 'http://192.168.99.100:4000';
+export const API_ENDPOINT = 'http://localhost:4000';
```
Comment (serpient) [inline main-app/client/src/get_uri.js]: putting localhost as an endpoint actually makes it fails on my Windows laptop.

@mitchellwandrews  @tienyuan  How do you think we should structure this so we can toggle between the Windows and non-Windows environments?

Comment (mitchellwandrews) [reply main-app/client/src/get_uri.js]: @serpient if you use 127.0.0.1 does that work?

Comment (serpient) [reply main-app/client/src/get_uri.js]: @mitchellwandrews `127.0.0.1` returns errors

Comment (tienyuan) [reply main-app/client/src/get_uri.js]: Maybe it can be bumped into a env variable that can be read from the docker config?

**Pull request merged**